Donut/__init__.py

```diff
@@ -0,0 +1,36 @@
+from flask import Flask, g
+from sqlalchemy import create_engine
+
+from Donut import config, constants
+from Donut.modules import example
+
+app = Flask(__name__)
+app.debug = False
+app.secret_key = config.SECRET_KEY
+
+# Maximum file upload size, in bytes.
+app.config['MAX_CONTENT_LENGTH'] = constants.MAX_CONTENT_LENGTH
+
+# Load blueprint modules
+app.register_blueprint(example.blueprint, url_prefix='/example')
+
+# Create database engine object.
+# TODO##DatabaseWork: We currently don't have a database set up, so we can't
+# reference sqlalchemy yet. However, it serves as a good example implementation.
+# engine = create_engine(config.DB_URI, convert_unicode=True)
+
+@app.before_request
+def before_request():
+  """Logic executed before request is processed."""
```

Comment:
Agreed. We'll probably have to change the style somewhat once we add documentation generation, since we'll have to add in more details (e.g. arguments, return). But I'll add the details in once I set that up.